front/app/containers/ProjectsShowPage/shared/events/index.tsx

```diff
@@ -58,4 +111,38 @@ const EventsContainer = memo<Props>(({ projectId, className }) => {
   return null;
 });
 
-export default EventsContainer;
+export default (props: InputProps) => {
+  const { projectId } = props;
+  const project = useProject({ projectId });
+  const [selectedPhase, setSelectedPhase] = useState<IPhaseData | null>(null);
+
+  useEffect(() => {
+    const subscription = selectedPhase$.subscribe((selectedPhase) => {
+      setSelectedPhase(selectedPhase);
+    });
+
+    return () => subscription.unsubscribe();
+  }, []);
+
+  if (isNilOrError(project)) return null;
+
+  const sort =
```

Comment:
Just a suggestion : try to centralize this, maybe in the project service directly or in the back-end somehow so that this order is enforced in the same way whenever necessary and could be changed with reasonable effort. (hypothesis : getting ideas for a project is done in so many places this would be worth the overhead)

Reply:
I am not sure how to add it in the service, since the service is just returning a stream... I guess I can try adding it to the backend

Reply:
Maybe a getSort utility in the project service, and/or maybe return that prop directly in the projects resources and hooks but that's already a fair amount of changes, and could set precedent for many other things we might want to have in there... the back-end solution might not be so straghtforward and set precendent too. Up to you, but a small tech debt item or CDM point would also solve for now